src/applets/budgie-menu/BudgieMenuWindow.vala

```diff
@@ -364,7 +364,7 @@ public class BudgieMenuWindow : Budgie.Popover
         right_layout.pack_start(content_scroll, true, true, 0);
 
         // placeholder in case of no results
-        var placeholder = new Gtk.Label("<big>Sorry, no items found</big>");
+        var placeholder = new Gtk.Label(_("<big>Sorry, no items found</big>"));
```

Comment:
IMO it should be `"<big>" + _("Sorry, no items found") + "</big>"` since `<big>` isn't something that needs to be translated. Not sure @ikeydoherty's thoughts.